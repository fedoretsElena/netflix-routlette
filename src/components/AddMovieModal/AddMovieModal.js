import React from "react";
import { Modal } from "react-bootstrap";
import { PropTypes } from "prop-types";

import MovieForm from './../MovieForm/MovieForm';

export default function AddMovieModal({show, handleClose, handleSubmit}) {
  return (
    <Modal id="add-movie-modal" show={show} onHide={handleClose} backdrop="static">
      <Modal.Header closeButton>
        <Modal.Title>Add movie</Modal.Title>
      </Modal.Header>

      <Modal.Body>
        <MovieForm handleSubmit={handleSubmit}/>
      </Modal.Body>
    </Modal>
  )
}

AddMovieModal.propTypes = {
  show: PropTypes.bool,
  handleClose: PropTypes.func,
  handleSubmit: PropTypes.func
}