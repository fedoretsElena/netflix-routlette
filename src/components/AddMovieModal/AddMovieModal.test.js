import React from 'react';
import { render } from '@testing-library/react';

import AddMovieModal from './AddMovieModal';

describe('AddMovieModal component', () => {
  it('should render AddMovieModal snapshot', () => {
    const { asFragment } = render(<AddMovieModal/>);
    expect(asFragment(<AddMovieModal/>)).toMatchSnapshot();
  })
})