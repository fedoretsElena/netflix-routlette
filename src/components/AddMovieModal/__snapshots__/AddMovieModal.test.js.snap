// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddMovieModal component should render AddMovieModal snapshot 1`] = `<DocumentFragment />`;
